keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -753,6 +798,21 @@ when drafting this test plan.
 
 ### Graduation Criteria
 
+#### Alpha -> Beta Graduation
+
+- A detailed DNS spec for multi-cluster services.
+- NetworkPolicy either solved or explicitly ruled out.
+- API group chosen and approved.
+- Kube-proxy can consume ServiceImport and EndpointSlice.
+- E2E tests exist for MCS services.
+- Beta -> GA Graduation criteria defined.
```

Comment:
Also: Formalization of cluster ID and plan for cluster ID implementation

Reply:
Added "a plan" to alpha -> beta, and "formal definition" to beta -> GA